datalad/cmd.py

```diff
@@ -181,7 +181,7 @@ def run(self, cmd, log_stdout=True, log_stderr=True,
                                     shell=shell,
                                     cwd=cwd)
             except Exception, e:
-                lgr.error("Failed to start %s: %s" % (cmd, e))
+                lgr.error("Failed to start %r: %s" % (cmd, " under %r" % cwd if cwd else '', e))
```

Comment:
Right.
This PR is just to make clear what I was talking about in #90. So, the only question is: Do you agree with that behaviour of only throwing exceptions in case Popen does and returning exit code otherwise?